Source/UnitTests/Common/MathUtilTest.cpp

```diff
@@ -33,8 +33,8 @@ TEST(MathUtil, IntLog2)
 
 TEST(MathUtil, NextPowerOf2)
 {
-  EXPECT_EQ(4, MathUtil::NextPowerOf2(3));
-  EXPECT_EQ(4, MathUtil::NextPowerOf2(4));
-  EXPECT_EQ(8, MathUtil::NextPowerOf2(6));
-  EXPECT_EQ(0x40000000, MathUtil::NextPowerOf2(0x23456789));
+  EXPECT_EQ(4u, MathUtil::NextPowerOf2(3));
```

Comment:
These should be upper-cased to be consistent relative to everywhere else in the codebase.